Week2/homework2/part2trigger.sql

```diff
@@ -0,0 +1,17 @@
+DELIMITER //
+CREATE TRIGGER mustafa_trigger
+    BEFORE INSERT
+        ON countrylanguage
+            FOR EACH ROW
+            BEGIN
+                DECLARE message VARCHAR(100);
+                DECLARE number0fLanguages INT;
+                SELECT COUNT(Language) INTO numberOfLanguages FROM countrylanguage WHERE country.countrycode=new.countrycode;
```

Comment:
small typo here between number0fLanguages and  numberOfLanguages

Reply:
Ok:)